keras_cv/models/backbones/mix_transformer/mix_transformer_backbone_presets.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MiT model preset configurations."""
+
+backbone_presets_no_weights = {
+    "MiT_B0": {
+        "metadata": {
+            "description": (
+                "MiT (MixTransformer) model with 8 transformer blocks."
+            ),
+            "params": 3321962,
+            "official_name": "MiT",
+            "path": "mit",
+        },
+        "class_name": "keras_cv.models>MiTBackbone",
+        "config": {
+            "embedding_dims": [32, 64, 160, 256],
```

Comment:
s/embedding_dims/embed_dims (or rename the arg in the backbone)

Probably it's more clear to keep it as embedding_dims and rename the arg.

Reply:
Good point - I'll rename the arg